doc/rados/operations/stretch-mode.rst

```diff
@@ -181,8 +185,8 @@ your CRUSH map. This procedure shows how to do this.
              step emit
      }
 
-   .. warning:: If a CRUSH rule is defined for a stretch mode cluster and the
-      rule has multiple "takes" in it, then ``MAX AVAIL`` for the pools
+   .. warning:: If a CRUSH rule is defined in stretch mode cluster and the
```

Comment:
s/in stretch mode cluster/in a stretch mode cluster/

Otherwise, this part of the docs sounds like Queen's "Ogre Battle".